Tests/KeyboardToolbarTests.swift

```diff
@@ -11,120 +11,112 @@ import XCTest
 
 class KeyboardToolbarTests: XCTestCase {
     
-    func test_KeyboardToolbar_InitWithFrame() {
-        let keyboardToolbar = KeyboardToolbar()
+    func test_initWithFrame() {
+        let frame = CGRect(x: 0, y: 0, width: 50, height: 50)
+        let keyboardToolbar = KeyboardToolbar(frame: frame)
         
         XCTAssertNotNil(keyboardToolbar)
-        XCTAssertEqual(keyboardToolbar.frame.height, 50)
+        XCTAssertEqual(keyboardToolbar.frame.height, frame.height)
```

Comment:
Would it be worth asserting equality on the `frame` itself instead of just `frame.height`?

Reply:
That's reasonable. I'll change it.